templates/protocols/bgp/node.def

```diff
@@ -10,6 +10,7 @@ syntax:expression: $VAR(@) >= 1 && $VAR(@) <= 4294967294 ; \
 end: if [ -z "$VAR(.)" ] || [ "$COMMIT_ACTION" != DELETE ]; then
        /opt/vyatta/sbin/vyatta-bgp.pl --main
        vtysh -d bgpd -c 'sh run' > /opt/vyatta/etc/quagga/bgpd.conf
+       sudo vtysh -n -w
```

Comment:
Can we use long options to make this code self-explanatory, like `vtysh --writeconfig`?